docs/howto.rst

```diff
@@ -259,23 +271,25 @@ By looking at this figure, we can work out that we need a smaller threshold in o
 
```

Comment:
It should say "middle" instead of "first" -  "we can work out that we need a smaller threshold in order to include this **middle** time point."